src/zenml/models/v2/core/artifact_visualization.py

```diff
@@ -105,6 +117,23 @@ def artifact_version_id(self) -> UUID:
         """
         return self.get_metadata().artifact_version_id
 
+    @property
+    def artifact_version(self) -> "ArtifactVersionResponse":
```

Comment:
The return of this is `Optional`, as the visualization might not have a linked artifact version in the DB.